### ISSUE TITLE
Chore: Licensing

### ISSUE THREAD

**Issue by kasperisager** [act-rules/act-rules.github.io#166]
I'm having a hard time determining the license under which Auto-WCAG rules are released. Without explicit licensing terms, I'm guessing it won't be possible for people to build upon exiting Auto-WCAG rules in order to incorporate them into their own set of accessibility rules?

**Comment by WilcoFiers** [comment]
Auto-WCAG operates under the W3C's community group license:
https://www.w3.org/community/about/agreements/cla/

Do you want to create a PR to include the license in this repo as well?

**Comment by kasperisager** [comment]
That's the CLA to which contributions are subject to though, and not the license under which the project itself is released; that's the part I'm missing. From the CLA:

> Any source code created by the Project is not subject to this CLA, but rather subject to separate licensing terms for that source code.

The "separate licensing terms for that source code" for the Auto-WCAG project does not seem to be stated anywhere.

**Comment by WilcoFiers** [comment]
I wouldn't qualify what we're doing as source code - but I guess that is a bit debatable. I'll reach out to Shadi, see if he has recommendations. We'll need to know this now that we're about to start moving work into the W3C space.

**Comment by kasperisager** [comment]
I'm fairly certain that the Berne Convention kicks in regardless of the definition though, so the rights to recite and adapt the rules would be exclusive to the authors unless stated otherwise.

**Comment by EmmaJP** [comment]
This might be helpful ... https://choosealicense.com/

**Comment by jkodu** [comment]
Merged. License can be viewed here - https://auto-wcag.github.io/auto-wcag/license.html

**Comment by WilcoFiers** [comment]
@nitedog WDYT?

**Comment by WilcoFiers** [comment]
When moving to act-cg we'll add a license file.

**Comment by jkodu** [comment]
Changes to licensing have been made - https://github.com/act-rules/act-rules.github.io/pull/600

And also a CLA assistant is set up for contributions.
